public_suffix_list.dat

```diff
@@ -5683,6 +5683,7 @@ int.ru
 net.ru
 org.ru
 pp.ru
+ru.net
```

Comment:
Note: This does not belong in this section, because it's not part of the .ru ccTLD. Instead, NIC.ru is selling subdomains from its own registration.

As such, NIC.ru would need to submit this change, to the PRIVATE section of the PSL.